//= output "Exception\n"

fun main() {
  let e = Exception();
  e.printStackTrace();
}
